bloom-housing/bloom
# feat: new applications release modal

This PR addresses #4189


## Description

You now cannot release a lottery to partners if you have added or updated application data since the last run.

## How Can This Be Tested/Reviewed?

As an admin on a closed listing with a lottery, run the lottery. Add or update an application on that listing, and then try to release the lottery. You should get [this modal](https://docs.google.com/document/d/130OuxoQnKnukoVr_SPQq57A0JZ4vycdcS9q0EX2Gkso/edit?pli=1#heading=h.w2usczb0e7mi), though with just one `OK` button and the additional text about updating.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
